File: SW Expert Academy/D2/01970. 쉬운 거스름돈/쉬운 거스름돈.py
money = [50000, 10000, 5000, 1000, 500, 100, 50, 10]
for tc in range(int(input())):
    N = int(input())

    print("#%d"%(tc+1))

    for m in money:
        print(N//m, end=' ')
        N %= m
    print()

File: SW Expert Academy/D2/01970. 쉬운 거스름돈/쉬운 거스름돈.cpp
#include <iostream>
#include <vector>
using namespace std;

int main(){
	int T; cin>>T;
	vector<int> money = {50000, 10000, 5000, 1000, 500, 100, 50, 10};
	for(int tc=0; tc<T; tc++){
		int N; cin>>N;
		
		printf("#%d\n", tc+1);
		
		for(int i=0; i<money.size(); i++){
			printf("%d ", N/money[i]);
			N %= money[i];
		}
        cout<<endl;
	}
}
